config/config.yaml

```diff
@@ -17,12 +17,25 @@ plank:
         entrypoint: "gcr.io/k8s-prow/entrypoint:v20230329-c93d79fb7d"
         sidecar: "gcr.io/k8s-prow/sidecar:v20230329-c93d79fb7d"
       gcs_configuration:
-        bucket: "jetstack-logs"
+        bucket: gs://jetstack-logs
         path_strategy: "legacy"
         # Leave this as k/k so that all logs use org_repo in the path
         default_org: "kubernetes"
         default_repo: "kubernetes"
       gcs_credentials_secret: "service-account"
+      resources:
+        clonerefs:
+          requests:
+            cpu: 100m
+        initupload:
+          requests:
+            cpu: 100m
+        place_entrypoint:
+          requests:
+            cpu: 100m
+        sidecar:
+          requests:
+            cpu: 100m
```

Comment:
Are these copied from upstream or are on basis of how much resources we need? And how does it compare with what we're requesting now?

Reply:
Currently, there are no requests for those init containers:
from the pod's yaml:
```yaml
resources: {}
```

These requests were copied from upstream: https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L26-L38